lms/djangoapps/labster_vouchers/views.py

```diff
@@ -118,6 +121,15 @@ def activate_voucher(request):
 
     course_license = course_licenses[0]
     course_id = course_license.course_id
+
+    # check invitation_only on course_overview is updated
+    course_key = SlashSeparatedCourseKey.from_deprecated_string(unicode(course_id))
```

Comment:
Ahh, I see so it isn't common situation. Yes agree with it but isn't better to update `invitation_only` through admin panel  i.e: https://staging-studio-courses.labster.com/admin/course_overviews/courseoverview/ ?